### PR TITLE
Update

### PR DESCRIPTION
**Pull request by RDIL**
## Description  
Quick update

## Checklist  

- [x] All *new* Java and Kotlin files have the right copyright header  
- [x] I have tested my code by building it locally  
- [x] This is not a work-in-progress and is ready for merge  
- [x] I am submitting the PR under and understand the terms of the GNU Lesser General Public License v3.0

### CODE REVIEW

**Comment by asbyth** [comment]
> already removed victory royale debug in my pr
> "dont worry about it, dont wanna have to deal with merge conflicts"
> does it anyway

**Comment by RDIL** [comment]
I didn’t realize you already did it @asbyth , my apologies.

**Comment by RDIL** [comment]
( @CoalCoding )

File: README.md [hunk 1]
```diff
@@ -12,46 +12,42 @@
...
+- [x] [NoNameTags](https://github.com/WesJD/NoNameTags)
+- [x] [Performia](https://github.com/Sk1er/Performia)
+- [x] [Perspective Mod](https://www.youtube.com/watch?v=7FdMMpzNdUk)
+- [x] [Resource Exploit Fix](https://github.com/Sk1er/Resource-Exploit-Fix) 
+- [x] ShinyPots
+- [x] [Sidebar Mod](https://www.youtube.com/watch?v=cn9VvT43yRs)
+- [x] [TimeChanger](https://www.youtube.com/watch?v=PbhXIPecOSA)
+- [x] [ToggleChat](https://www.youtube.com/watch?v=guD8kAk-Wn4)
+- [x] [Victory Royale Mod](https://www.youtube.com/watch?v=g4pZTX8ooqI)
+- [x] [VoidFlickerFix](https://www.youtube.com/watch?v=klV4d1B6ysk)
```
[the remark this answers is not in the document]
Comment (RDIL) [reply README.md]: I was just inserting this as more of a cosmetic benefit, almost as to say we have it all.  I will remove it if you want.

File: src/main/java/cc/hyperium/handlers/handlers/HypixelDetector.java [hunk 2]
```diff
@@ -52,6 +53,7 @@ public HypixelDetector() {
         instance = this;
     }
 
+    @Contract(pure = true)
     public static HypixelDetector getInstance() {
```
[the remark this answers is not in the document]
Comment (RDIL) [reply src/main/java/cc/hyperium/handlers/handlers/HypixelDetector.java]: IntelliJ told me to put them here so I did

File: src/main/java/cc/hyperium/purchases/PurchaseApi.java [hunk 3]
```diff
@@ -131,7 +131,10 @@ public HyperiumPurchase getPackageIfReady(UUID uuid) {
     }
 
     public void getPackageAsync(UUID uuid, Consumer<HyperiumPurchase> callback) {
-        Multithreading.runAsync(() -> callback.accept(getPackageSync(uuid)));
+        try {
+            Multithreading.runAsync(() -> callback.accept(getPackageSync(uuid)));
+        } catch (NullPointerException e) {
+        }
     }
```
[the remark this answers is not in the document]
Comment (RDIL) [reply src/main/java/cc/hyperium/purchases/PurchaseApi.java]: Okay

File: src/main/java/cc/hyperium/handlers/handlers/HypixelDetector.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (RDIL) [reply src/main/java/cc/hyperium/handlers/handlers/HypixelDetector.java]: I will remove them when I get home

**Comment by RDIL** [comment]
Thanks Coal :)

**Pull request merged**